src/components/Popups/ReplyField/PopupList.tsx

```diff
@@ -28,12 +36,18 @@ const options: Partial<Options> = {
     placement: 'bottom-start',
 };
 
-const PopupList = ({ reference, ...rest }: Props): JSX.Element => (
-    <PopupBase className="ba-PopupList" options={options} reference={reference} {...rest}>
-        <div className="ba-PopupList-prompt ba-PopupList-item">
-            <FormattedMessage {...messages.popupListPrompt} />
-        </div>
-    </PopupBase>
-);
+const PopupList = ({ items, reference, options, ...rest }: Props): JSX.Element => {
+    return (
+        <PopupBase className="ba-PopupList" options={{ ...defaultOptions, ...options }} reference={reference}>
```

Comment:
This will probably cause the popper to be re-rendered constantly due to the object reference changing per-render.

Reply:
What was the finding here?

Reply:
> What was the finding here?

I feel I eventually need to pass in options in the future. But since it's not used in this pr, I will remove it.